pkgs/desktops/elementary/apps/switchboard-plugs/pantheon-shell/default.nix

```diff
@@ -0,0 +1,54 @@
+{ stdenv, fetchFromGitHub, elementary, meson, ninja, pkgconfig, vala
+, libgee, granite, gexiv2, elementary-settings-daemon, gtk3, gnome-desktop
+, plank, switchboard, gettext, gobject-introspection, bamf }:
+
+stdenv.mkDerivation rec {
+  pname = "switchboard-plug-pantheon-shell";
+  version = "2.8.0";
+
+  src = fetchFromGitHub {
+    owner = "elementary";
+    repo = pname;
+    rev = version;
+    sha256 = "0yy821hl26jfd9hyigqi7nmaf30iww0lhg9qzcwlfzsvvfwnxagi";
+  };
+
+  passthru = {
+    updateScript = elementary.updateScript {
+      repoName = pname;
+    };
+  };
+
+  nativeBuildInputs = [
+    gettext
+    gobject-introspection
+    meson
+    ninja
+    pkgconfig
+    vala
+  ];
+
+  buildInputs = [
+    bamf
+    gexiv2
+    gnome-desktop
+    elementary-settings-daemon
+    granite
+    gtk3
+    libgee
+    plank
+    switchboard
+  ];
```

Comment:
I do not think this will be enough, as the package produces a library which will not be wrapped.

Reply:
Ugh, the schema glory https://github.com/elementary/switchboard-plug-pantheon-shell/blame/91b5b9b2f2ae614c1c2e88490e9c67d431431777/src/Views/Appearance.vala#L25

I could add these to `switchboard`.

Reply:
I finally formulated some sort of GSettings policy: https://github.com/NixOS/nixpkgs/pull/54083#issuecomment-455156084

It would be best to fix this in the plug-in code.

Reply:
You mean by patching the source hardcoding the gsettings?

Reply:
Yup.

Reply:
Done.